tox.ini

```diff
@@ -0,0 +1,35 @@
+[tox]
+requires =
+    tox-conda
+
+[testenv]
+conda_spec =
+    py38: {toxinidir}{/}requirements{/}locks{/}py38-linux-64.lock
+    py39: {toxinidir}{/}requirements{/}locks{/}py39-linux-64.lock
+    py310: {toxinidir}{/}requirements{/}locks{/}py310-linux-64.lock
+usedevelop =
+    true
```

Comment:
These are all inherited by the below test environments, although note that with the wheel job `skip_install` takes precedence over `usedevelop`.